packages/web-components/fast-foundation/src/text-field/text-field.ts

```diff
@@ -204,9 +204,11 @@ export class TextField extends FormAssociated<HTMLInputElement> {
 
         this.proxy.setAttribute("type", this.type);
 
-        if (this.autofocus) {
-            this.focus();
-        }
+        DOM.queueUpdate(() => {
```

Comment:
Can you check autofocus before queuing an update?  I don't think we want all text fields queing something.

Reply:
+1 - would suggest we only do this in the case autofocus is true.

Reply:
this makes sense, will change

Reply:
I'm assuming nothing can change with regard to this.autofocus between checking and queuing the update, so only 1 check before the queue call